02 - Memtables, Joins, and Selectors.ipynb

```diff
@@ -463,7 +463,7 @@
    "outputs": [],
```

Comment:
<p>I had to restart the notebook server to be able to run this. Restarting just the kernel didn't work. </p><p> </p><p>Not sure if this is a bug, or what, but I had the same issue you did on you EuroScipy tutorial (saw recording)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>